src/main/java/com/google/ehub/utility/Utils.java

```diff
@@ -59,4 +62,24 @@ public static String getNextHigherString(String str) {
           + str.substring(lastNonMaxCharIndex + 1);
     }
   }
+
+  /**
+   * Gets the millisecond timestamp of a given date in a specific format
+   * @param date the date to convert into a timestamp eg. ("04-23-2020")
+   * @param format the format used by the given date eg. ("MM-DD-YYYY")
+   * @return the timestamp of the date in milliseconds, or null if it failed to parse
+   */
+  public static Long getTimestampMillisFromDate(String date, String format) {
+    SimpleDateFormat dateParser = new SimpleDateFormat(format);
+    Date parsedDate;
+
+    try {
+      parsedDate = dateParser.parse(date);
+    } catch(ParseException e) {
+      System.err.println("Failed to parse date!");
```

Comment:
lets also print the error in case it has useful info: `System.err.println("Failed to parse date with error " + e);`

Reply:
I added the exception info in the print statement in last commit.